src/Tizen.NUI.Components/Theme/DefaultTheme.cs

```diff
@@ -128,6 +128,20 @@ public HashSet<ExternalThemeKeyList> GetExternalThemeKeyListSet()
                     .Add<Size>("/BottomLine/Size", (ViewStyle style, Size value) => ((TabButtonStyle)style).BottomLine.Size = value)
                     .Add<Position>("/BottomLine/Position", (ViewStyle style, Position value) => ((TabButtonStyle)style).BottomLine.Position = value)
                     .AddSelector<Color>("/BottomLine/BackgroundColor", (ViewStyle style, Selector<Color> value) => ((TabButtonStyle)style).BottomLine.BackgroundColor = value, ControlState.Selected),
+                
+                // TimePicker
+                (new ExternalThemeKeyList(typeof(TimePicker), typeof(TimePickerStyle)))
+                    .Add<Size>("/CellPadding", (ViewStyle style, Size value) => ((ViewStyle)style).Size = value)
+                    .Add<Size>("/Pickers/Size", (ViewStyle style, Size value) => ((TimePickerStyle)style).Pickers.Size = value)
```

Comment:
Please change this as follows.

```
((TimePickerStyle)style).Pickers.Size = value)                 ->                ((PickerStyle)style).Pickers.Size = value)
```

Reply:
Pickers is a PickerStyle, so isn't it correct that the style to be referenced is a TimePicker?